arbeitszeit_flask/database/repositories.py

```diff
@@ -731,6 +731,15 @@ def add_worker(self, member: UUID) -> int:
             company.workers.append(member)
         return companies_changed
```

Comment:
Sorry to  hold things up, but do we expect ever to use this return value?  It looks as if we never do for add_worker — e.g., in arbeitszeit/use_cases/answer_company_work_invite.py, we have this:
`            self.database_gateway.get_companies().with_id(invite.company).add_worker(invite.member)`
It does not hurt, of course.

Reply:
Yes, it seems to me that we do not use it currently. But as you said, it does not hurt..